src/operator/tensor/indexing_op.cu

```diff
@@ -209,6 +240,9 @@ NNVM_REGISTER_OP(gather_nd)
 NNVM_REGISTER_OP(scatter_nd)
 .set_attr<FCompute>("FCompute<gpu>", ScatterNDForward<gpu>);
 
+NNVM_REGISTER_OP(scatter_nd_acc)
```

Comment:
The string `acc` looks ambiguous. I thought it standed for `accurate` in the beginning, but realized that it means `accumulate` later. It's named `scatter_nd_add` in TF, as there are also `scatter_nd_sub`, `scatter_nd_mul`, and `scatter_nd_div`. Shall we also call it `scatter_nd_add` to be precise?

Reply:
There is a slight difference between `scatter_nd_add` and `scatter_nd_acc`. In scatter_nd_add, the results are added to another array. While in scatter_nd_acc, the values are added to a all-zero array. The number of arguments are different for these two OPs.